src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -490,9 +490,11 @@ protected final void verify(Checker checker,
             throws Exception {
         stream.flush();
         stream.reset();
-        final List<File> theFiles = new ArrayList<>();
-        Collections.addAll(theFiles, processedFiles);
```

Comment:
`theFiles` is kind of strange name anyway, as `the` just like `my` is not a good prefix for a field as its pretty generic and therefore adding fluff. its just the `files` and not need to couple this. just give the value.

Simple `List.of(` usage should be applied anyway, so lets just elevate this as we already on it.
`List<File> files = List.of(processedFiles);`